cost-model/src/transaction_cost.rs

```diff
@@ -24,10 +24,10 @@ impl TransactionCost {
         }
     }
 
-    pub fn bpf_execution_cost(&self) -> u64 {
+    pub fn programs_execution_cost(&self) -> u64 {
         match self {
-            Self::SimpleVote { .. } => 0,
-            Self::Transaction(usage_cost) => usage_cost.bpf_execution_cost,
+            Self::SimpleVote { .. } => solana_vote_program::vote_processor::DEFAULT_COMPUTE_UNITS,
```

Comment:
above:
```rust
const SIMPLE_VOTE_USAGE_COST: u64 = 3428;
```

Is the `TransactionCost` of simple votes now self-consistent? i.e. is `sum() == programs_execution_cost() + (others?)` now?

Reply:
if this is something, it's still fine to have the return from `sum` to be a constant instead of actually summing, but we should have some checks that returned sum is the sum of parts I think

Reply:
for `is_simple_vote`, it's cost is const, mentioned in comments above and in #33269. It is `sum() == programs_execution_cost() + loaded_accounts_data_size_cost() + signature_cost() + write_lock_cost()`

Probably can do a const assertion within `sum()` like: 
```
fn sum(&self) -> u64 {
    const _: () = assert!(SIMPLE_VOTE_USAGE_COST == solana_vote_program::vote_processor::DEFAULT_COMPUTE_UNITS + block_cost_limits::SIGNATURE_COST + block_cost_limits::WRITE_LOCK_UNITS * 2 + 8 );
}
```
in a separate pr?